single-sign-on/index.md

```diff
@@ -35,20 +39,40 @@ To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank
 
 1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to Organizations and select the organization that you want to enable SSO on.
 2. Click **Settings** and select the Security tab.
-3. Select an authentication method based on your identity provider.
-    Note: Docker currently supports **SAML 2.0** and **Azure AD**.
+3. Select an authentication method based on your identity provider. Docker currently supports **SAML 2.0** and **Azure AD**.
 4. Copy the ID and/or URL in the **Identity Provider Set Up**.
-    Note: for SAML 2.0, copy the Entity ID and ACS URL. For Azure AD, copy your Redirect URL/Reply URL.
+    For SAML 2.0, copy the **Entity ID** and **ACS URL**. For Azure AD, copy your **Redirect URL/Reply URL**.
 5. Log into your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
-6. Complete the fields in the **Configuration Settings** section and click **Save**.
+6. Complete the fields in the **Configuration Settings** section and click **Save**. If you want to change your IdP, you must delete your existing provider and configure SSO with your new IdP.
 
 ![SSO SAML](images/sso-saml.png){:width="500px"}
 
-### Domain
+### Domain control
 
-1. Click **Add Domain** and specify the email domains that are allowed to authenticate via your server.
-    Note: This should include all email domains users will use to access Docker. Public domains are not permitted, such as gmail.com, outlook.com, etc. Also, the email domain should be set as the primary email.
-2. Click **Send Verification** to receive an email for the domains you have specified and verify your domain.
+Click Add Domain and specify the corporate domain you’d like to manage with SSO. Domains should be formatted without protocol or www information, for example, yourcompany.com.
```

Comment:
I'm wondering if we should include something describing that the domain they're looking for is the domain that will be used by their users as part of their e-mail. Perhaps that makes it easier to understand (companies may have various domains, e.g., Docker has `docker.io`, `dckr.io`, and `docker.com`, but (I think) only `docker.com` is relevant in this case as that's the domain used for e-mail addresses that employees use.

That way we could describe that if users will use `user@company-mail.org`, then they need to use that domain (everything after `@`).


I'm not sure though what needs to be entered if (less common) their users use e-mail addresses on subdomains. For example, I know IBM used per-department subdomains at some point (`name@linux.vnet.ibm.com`). Do they need to enter `ibm.com` in that case, or `linux.vnet.ibm.com` ?

Let me try get an answer to that last one ^^

Reply:
@thaJeztah in line 56 we mention the users and access to Docker and the domains that aren't permitted. I can check to see if there are other instances where they would use a domain such as:  linux.vnet.ibm.com